api/lanes.py

```diff
@@ -1242,3 +1242,82 @@ def initialize(self, library, customlist):
     def url_arguments(self):
         kwargs = dict(list_name=self.customlist_name)
         return self.ROUTE, kwargs
+
+class KnownOverviewFacetsWorkList(WorkList):
+    """A WorkList whose defining feature is that the Facets object
+    to be used when generating a grouped feed is known in advance.
+    """
+    def __init__(self, facets, *args, **kwargs):
+        """Constructor.
+
+        :param facets: A Facets object to be used when generating a grouped
+           feed.
+        """
+        super(KnownOverviewFacetsWorkList, self).__init__(*args, **kwargs)
+        self.facets = facets
+
+    def overview_facets(self, _db, facets):
+        """Return the faceting object to be used when generating a grouped
+        feed.
+        
+        :param _db: Ignored -- only present for API compatibility.
+        :param facets: Ignored -- only present for API compatibility.
+        """
+        return self.facets
+
+
+class JackpotWorkList(WorkList):
```

Comment:
@EdwinGuzman I moved this in from `core` because it's only used by the circulation manager.

Reply:
Does this mean that https://github.com/NYPL-Simplified/server_core/pull/1198/ can be closed?

Reply:
It sure does!